### PR TITLE
FS :  Friends polls page

### LINKED ISSUE #6: FE: Friends Polls Page
- User can see polls they were invited to, in the order they were invited

### PR DESCRIPTION
**Pull request by aalthaf**
- User can see the polls they were invited to in order

- Each Poll has a link to PollView where the user can vote

- New controller for invited polls 

- Fixed undefined error which happens when user tries to access a poll with total of zero votes ( in PollView )

### CODE REVIEW

File: client/src/components/FriendsPollsPage/FriendsPollsPage.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  const [polls, setPolls] = React.useState("");
+  const classes = useStyles();
+  React.useEffect(() => {
+    getInvitedPolls().then((polls) => setPolls(polls));
+  }, []);
+
+  return (
+    <div className={classes.root}>
+      <h2>You were invited to the following polls</h2>
+      <Box p={1} pl={25} pr={26}>
```
Comment (yoniakabecky) [inline client/src/components/FriendsPollsPage/FriendsPollsPage.js]: I think `Box` is causing a lot of side padding, how about using `Container` instead?

File: client/src/components/FriendsPollsPage/FriendsPollsPage.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <Divider className={classes.divider} />
+        <div>
```
Comment (yoniakabecky) [inline client/src/components/FriendsPollsPage/FriendsPollsPage.js:43]: Is this `div` necessary?

File: client/src/pages/FriendsPolls.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import DashboardLayout from "../components/DashboardLayout";
+import FriendsPollsPage from "../components/FriendsPollsPage/FriendsPollsPage";
+import { NAVBAR_HEIGHT } from "../utils/constants";
+
+export default function Friends() {
```
Comment (yoniakabecky) [inline client/src/pages/FriendsPolls.js]: Function Friends already exist... 
Maybe better to match the function name and file name??

**Review by sina-jamshidi — APPROVED** [review]
Looks good!

**Pull request merged**